general_rules/Scenario.tex

```diff
@@ -127,9 +127,18 @@ \subsection{Objects}
 \paragraph*{Important note:} It is not allowed to modify any of the objects provided for training. Also, teams are not allowed to keep more than 5 the objects provided for training at a time nor retaining it for more than one hour.
 
 \subsubsection{Containers}
-The TC will provide at least two containers (a transport container such as a tray and a pouring container such as a bowl) which will be available for training during the setup days.
+The TC will provide at three different types of containers to be used in the tests.
 
-There are no restrictions on a container size, appearance or weight; however, it can be expected that the selected containers be lightweight, with handles, and easily manipulable by a human using both hands.
+\begin{itemize}
```

Comment:
For *Help-me-carry* there is no other container unless we fully incorporate the box (I will remove the reference in a PR later today). However, other tests (¿?) use other types of containers, such as:
- Bag
- Basket (you used one in German Open)
- Bowl
- Tray
- Plate

Bowl and plate are containers in the very moment you can pour milk/cereal and bread on them, respectively. Basket is useful for placing bread in it (e.g. serving the table), but it may also be used along with the tray to transport 2+ objects in a test like *Set the table...* and *Restaurant*. A smart robot could place all cutlery and dishes in the basket, transport the basket to the table, and do the setting with one single transport instead of 3+.